python/lsst/pipe/supertask/cmdLineFwk.py

```diff
@@ -161,7 +158,8 @@ def parseAndRun(self, argv=None):
             pipeline2dot(pipeline, args.pipeline_dot, self.taskFactory)
 
         if args.subcommand == "build":
-            # stop here
+            # stop here but process --show option first
+            self.showInfo(args.show, pipeline, None)
```

Comment:
`build` does useful things (producing pickled Pipeline ad DOT file) but --show option was ignored before because of `return`. I'd like to improve/simplify logic here, but that will be done as a part of big refactoring later.